python/Ganga/GPIDev/Credentials/ICredential.py

```diff
@@ -524,3 +524,4 @@ def timeleftInHMS(self, force_check=False):
         """
         logger.warning("Dummy method used - no information returned")
         return ""
+
```

Comment:
sorry, for a start new lines are picked up as `W292` warning suggesting they should be there and are added by some editors by default as good practice. I'll remove as not needed for this PR.